src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -14,8 +14,7 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-#import <Matter/MTRClusterConstants.h>
-#import <Matter/MTRDefines.h>
+#import <Matter/Matter.h>
```

Comment:
This is also not the correct import, this should be a precompiled header for the project, but this'll patch it over for now.